src/Momento.Sdk.Incubating/Responses/CacheDictionaryGetFieldResponse.cs

```diff
@@ -10,24 +13,34 @@ public abstract class CacheDictionaryGetFieldResponse
     public class Hit : CacheDictionaryGetFieldResponse
     {
         protected readonly ByteString value;
+        protected readonly ByteString field;
 
-        public Hit(_DictionaryGetResponse response)
+        public Hit(IEnumerable<ByteString> fields, _DictionaryGetResponse response)
```

Comment:
Curious why this signature accepts an array / enumerable instead of just the individual field?